zeebe/broker/src/main/java/io/camunda/zeebe/broker/system/configuration/ExportingCfg.java

```diff
@@ -7,43 +7,23 @@
  */
 package io.camunda.zeebe.broker.system.configuration;
 
-import java.util.Objects;
+import java.time.Duration;
 import java.util.Set;
 
 /**
  * Exporting component configuration. This configuration pertains to configurations that are common
  * to all exporters.
  */
-public final class ExportingCfg implements ConfigurationEntry {
-  private Set<Long> skipRecords;
+public record ExportingCfg(Set<Long> skipRecords, Duration distributionInterval) {
+  public static final Duration DEFAULT_DISTRIBUTION_INTERVAL = Duration.ofSeconds(15);
```

Comment:
:wrench: The default in `ExporterDirectorContext` is still there, it would be good if we could define a default once and then reference it from the other class.